dev-tools/mage/settings.go

```diff
@@ -98,6 +101,8 @@ func init() {
 	if err != nil {
 		panic(errors.Errorf("failed to parse SNAPSHOT env value", err))
 	}
+
+	versionQualifier, versionQualified = os.LookupEnv("BEAT_VERSION_QUALIFIER")
```

Comment:
From what I understand `versionQualified` is always == `versionQualifier != ""`. Would it make sense to remove that variable to avoid confusion?

Reply:
We can do that once the default build qualifier is empty.  See the description where `mage package` produces a binary where the `version` subcommand is different than that produced by `BEAT_VERSION_QUALIFIER= mage package`.

Reply:
We could do that immediately if we update release manager first.

Reply:
Got it, thanks for the clarification